proposals/stage-0/COLLAB_SUMMIT/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Collaborator Summit
+>  Stage 0
+
+## Champion
+
+Matteo Collina (@mcollina)
+
+## Description
+
+Organize an OpenJS Collaborator Summit twice a year, one in Europe and
+one in the US, following the lead of the [Node.js Collaborator Summit](https://github.com/nodejs/summit).
```

Comment:
Is "in the US" code here for "in North America", or does it intentionally exclude Canada?

Reply:
Considering that this year will be the third year in a row that the the late-in-the-calendar-year Collaborator Summit is in Canada and not the US, I'm guessing it should at least say North America rather than US, if not simply the Americas.

Given the large number of users and contributors we have in places like India and China (and Colombia and...), I wonder if the whole "once in Europe, once in North America" thing needs to be re-thought, or will need to be re-thought soon. Not blocking on it, but something to watch for sure.

Reply:
Perhaps it could be broadened to "Two geographically distinct locations"?

Reply:
Non-blocking and don't want to get too into the weeds on this, but: I wonder if this is all getting overly specific? Like, why specify twice a year? Maybe sometimes it will be three and sometimes one? Or maybe the people organizing it will find that having it occur at a consistent cadence is good, so twice a year always. But why tie their hands upfront?

Same with geographically distinct locations. Maybe include text that indicates that the timing and location should consider maximizing participation etc. In other words, explain the values that are driving the geographically distinct locations, but don't specify the actual practice. Leave that up to the implementors?

Reply:
(Regardless, "North America" is better than just "US", and "geographically distinct locations" is even better yet. Feel free to ignore my "wonder if this is all getting overly specific" comment above. It's certainly true that the document can always be modified later anyway, so we're not really tying people's hands perhaps.)